Website/structure-sources/index.rakudoc

```diff
@@ -19,24 +19,48 @@
         <div class="container px-4">
           <div class="columns is-multiline">
             <!-- Card -->
-            <div class="column is-one-third">
+            <div class="column is-one-half">
               <div class="card card-home">
                 <div class="card-content">
                   <div class="has-text-centered">
-                    <a href="/language">
+                    <a href="/introduction">
```

Comment:
We need to add a redirect for any old urls, like /language. I see the individual pages haven't moved, do we need to move this one?

Reply:
@coke I had not thought of inward pointing urls, eg. from someone's blog. There are two ways of handling this:
- Generate the 'old' language page, and include some wording in the rendering like 'this page is now deprecated, please change to 'introduction' or 'tutorial', and similarly for the 'program' page.
- Create a mapping file where the  `/language` route is rewritten to `/introduction` route. (respectively for 'program'). Caddy provides for such a mapping file, which I can create. This would serve the same purpose as the rewrite section in a `.htaccess` file for Apache.
  - There is already a rewrite file called 'tidy-urls' that is used by Caddy.
  - We could have another file called `deprecated-links` that uses the same Caddy functionality.

I think the second variation is better because when source files change names (which is likely to happen some time), we can then add an appropriate mapping to the `deprecated-links` file.

Reply:
Yes, let's do the mapping; we should avoid breaking old links whenever possible.